config.go

```diff
@@ -167,3 +167,14 @@ func argsStringFromMap(input map[string]string, currentArgs ...string) string {
 
 	return strings.ReplaceAll(params.Encode(), "&", ",")
 }
+
+// FeedStartFn function to set options on the [Client.NewEventFeed]
+type FeedStartFn func(req *feedRequest)
```

Comment:
Hmm, I advised toward a FeedStartFn because QueryOptFn was already being used as a variadic for the FeedFromQuery case and the FeedStartFn were mutually exclusive. See above comments.

At the very least the three of us should discuss this (probably live) so Darren isn't being pulled in opposing directions.